### PR TITLE
Update MoveNextGen.py

### PR DESCRIPTION
**Pull request by imvarungupta**
indentation corrected

### CODE REVIEW

File: MoveNextGen.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 with open ('uniqDll.txt', 'r') as Dl:
-	for line in Dl:
-		rec = line.strip()
-		NameWithoutDll , extra = rec.split('.dll')
-		print rec + "|" + NameWithoutDll + ";<.*>d__.*|MoveNext|0|0"
+    for line in Dl:
+        rec = line.strip()
+	NameWithoutDll , extra = rec.split('.dll')
+	print rec + "|" + NameWithoutDll + ";<.*>d__.*|MoveNext|0|0"
```
Comment (ShubhamRajvanshi10) [inline MoveNextGen.py]: these two lines should be in same level as "rec = line.strip()"

File: MoveNextGen.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import os
 with open ('uniqDll.txt', 'r') as Dl:
-	for line in Dl:
-		rec = line.strip()
-		NameWithoutDll , extra = rec.split('.dll')
-		print rec + "|" + NameWithoutDll + ";<.*>d__.*|MoveNext|0|0"
+    for line in Dl:
```
Comment (ShubhamRajvanshi10) [inline MoveNextGen.py:3]: indentation level should be 4 spaces

**Review by tarunrajvanshi — APPROVED** [review]
Done.

**Review by ShubhamRajvanshi10 — APPROVED** [review]
🚀

**Pull request merged**